db/migrate/20160808150745_add_metrics_status_to_ext_management_system.rb

```diff
@@ -0,0 +1,7 @@
+class AddMetricsStatusToExtManagementSystem < ActiveRecord::Migration[5.0]
+  def change
+    add_column :ext_management_systems, :last_metrics_error, :text
+    add_column :ext_management_systems, :last_metrics_update_date, :timestamp
+    add_column :ext_management_systems, :last_metrics_success_date, :timestamp
```

Comment:
@simon3z yeah ... man, I need to think about this one.

Lemme see if I can break this down a bit.  We currently have:

* `authentications.status`: stores the status of the authentication/connection
  - `authentications.status_details` has the details of the error, if there was one
* `ext_management_systems.last_refresh_date`: stores the date of the last ems refresh
  - `ext_management_systems.last_refresh_error` has the details of the last refresh failure, if there was one.

We don't currently store any type of status on the `endpoints` table.

Also, part of the problem is that we don't store anything in the DB specifying what ManageIQ Provider Capability each each endpoint is used for (i.e., Refresh, Events, Metrics).  That distinction could be captured just in the logic of each Manager.  But, if a single endpoint is used for multiple capabilities (e.g., a single endpoint used for Refresh *and* Metrics) there would need to be some way to indicate the different status values for each capability on that endpoint.

I don't see us figuring that out anytime soon, honestly.

For now it might be better to just add the metrics status columns to `ext_management_systems` and refactor the status tracking later.